src/commonMain/kotlin/fr/acinq/lightning/wire/LightningMessages.kt

```diff
@@ -1747,6 +1751,50 @@ data class PhoenixAndroidLegacyInfo(
     }
 }
 
+data class DNSAddressRequest(val offer: OfferTypes.Offer, val languageSubtype: String) : LightningMessage {
```

Comment:
The name can be confusing, so it's worth some documentation on that message to explain what it does? Also, I believe this should have a `chainHash` and extend `HasChainHash`.

Reply:
I can totally rename to `Bip353DNSAddressRequest` if you prefer?

Reply:
That's not really necessary, but I think it's important to document what this does, and that the peer may simply not respond.

Reply:
Done in ad3b2efa.